src/events/crawler/scrapers/USA/VA/index.js

```diff
@@ -150,54 +153,82 @@ const scraper = {
   ],
 
   async scraper() {
-    const pdfBaseURL = `${this.url}.pdf?:showVizHome=no&Locality=`;
-    const fullNameCounties = [
-      'Buena Vista city',
-      'Fairfax city',
-      'Franklin city',
-      'Franklin County',
-      'Manassas city',
-      'Richmond city',
-      'Richmond County',
-      'Roanoke city',
-      'Roanoke County'
-    ];
-    const city2County = ['Buena Vista city', 'Manassas city'];
-    const counties = [];
+    const usePDFs = datetime.scrapeDateIsBefore('2020-3-26');
+    this.url = usePDFs
+      ? 'https://public.tableau.com/views/VirginiaCOVID-19Dashboard/VirginiaCOVID-19Dashboard'
+      : 'http://www.vdh.virginia.gov/content/uploads/sites/182/2020/03/VDH-COVID-19-PublicUseDataset-Cases.csv';
+    let counties = [];
 
-    for (let name of this._counties) {
-      const endURL = fullNameCounties.includes(name) ? name : name.slice(0, name.lastIndexOf(' '));
-      const pdfUrl = pdfBaseURL + endURL;
-      const pdfScrape = await fetch.pdf(pdfUrl);
+    if (usePDFs) {
+      const pdfBaseURL = `${this.url}.pdf?:showVizHome=no&Locality=`;
+      const fullNameCounties = [
+        'Buena Vista County',
+        'Fairfax city',
+        'Franklin city',
+        'Franklin County',
+        'Manassas County',
+        'Richmond city',
+        'Richmond County',
+        'Roanoke city',
+        'Roanoke County'
+      ];
+      const county2City = ['Buena Vista County', 'Manassas County'];
 
-      if (city2County.includes(name)) {
-        name = name.replace('city', 'County');
-      }
+      for (const name of this._counties) {
+        let endURL = name;
+        if (county2City.includes(name)) {
+          endURL = endURL.replace('County', 'City');
+        }
+        if (!fullNameCounties.includes(name)) {
+          endURL = endURL.slice(0, name.lastIndexOf(' '));
+        }
+        const pdfUrl = pdfBaseURL + endURL;
+        const pdfScrape = await fetch.pdf(pdfUrl);
 
-      if (pdfScrape) {
-        let pdfText = '';
-        for (const item of pdfScrape) {
-          if (item.text === '©') {
-            break;
+        if (pdfScrape) {
+          let pdfText = '';
+          for (const item of pdfScrape) {
+            if (item.text === '©') {
+              break;
+            }
+            pdfText += item.text;
           }
-          pdfText += item.text;
+
+          counties.push({
+            county: name,
+            cases: parse.number(pdfText.match(/(\d*)Cases/)[1]),
+            deaths: parse.number(pdfText.match(/(\d*)Deaths/)[1])
+          });
+        } else {
+          counties.push({
+            county: name,
+            cases: 0
+          });
+        }
+      }
+    } else {
+      const cites = ['Fairfax City', 'Franklin City', 'Manassas City', 'Richmond City', 'Roanoke City'];
```

Comment:
```suggestion
      this.url = 'http://www.vdh.virginia.gov/content/uploads/sites/182/2020/03/VDH-COVID-19-PublicUseDataset-Cases.csv';
      this.type = 'csv';

      const cites = ['Fairfax City', 'Franklin City', 'Manassas City', 'Richmond City', 'Roanoke City'];
```

You have to set `this.url` and `this.type` if you ever guard by date, otherwise it will think we're using a PDF when we run the timeseries.